### ISSUE TITLE
[game mechanics] infected characters accumulate more burn damage

### ISSUE THREAD

**Issue by Oukido** [Regalis11/Barotrauma#8870]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
When testing the burn damage changes I noticed that characters with husk infection 90 (+husk genes) get more burn damage than characters without infection. I have not found any information about husks getting more burn damage, neither do the infected characters benefit of any healing abilities of husks.

I tested a flat 30 burn affliction (via console) on 2 engineer characters with one having husk and one without.
- both healed equally fast

But when giving both characters radiation sickness the infected character accumulates burn damage faster.
Tested with same clothes and without clothes.

When looking into the xml files I found this:
Human.xml:
<health ... BleedingReduction="0" BurnReduction="0.075" ConstantHealthRegeneration="0" ... >

Humanhusk.xml:
<health ... bleedingreduction="10" burnreduction="0" constanthealthregeneration="1" ... >

I'm not sure if "burnreduction" is effective but I also can't see the other values have any effect on infected characters.

**Version**
![grafik](https://user-images.githubusercontent.com/83960267/164326502-cfe82275-0662-4c92-813f-7010be07fa55.png)

**Comment by Regalis11** [comment]
I'm afraid I could not find a way to reproduce this. Both characters seem to be accumulating burns at the same rate:
![image](https://user-images.githubusercontent.com/14351070/164410262-c68c9cc2-d1f0-4886-916b-73bd376ddffd.png)

The humanhusk.xml is not actually relevant here: that's the config file of the AI-controlled husk the character would turn into if the infection reached 100%. Falling unconscious at a different level of burns would suggest that the characters still had a different amount of vitality for some reason: could it be that you got a vitality-reducing negative effect (e.g. Decrepify) from the gene splicing?

**Comment by Oukido** [comment]
Its not only husk genes, but also having husk infection at deep purple state (90)

**Comment by Oukido** [comment]
I think I found the reason:
I tested all the mutations that give the character appendages and everyone increased the burn received from radiation sickness.
Although the mudraptor beak only caused a marginal 1-2% damage difference.
But the husk appendage and the spineling appendage greatly increased the burn damage received.

My guess is that the additional limb is getting additional burn damage which is cumulated on the body part it is attached.
In theory that should be true for any kind of damage then. ( ? )

In my final test I gave one character all 3 appendages and did the test again and it felt unconcious at 100% burn damage where the other character without any appendages only had 64% burn damage.

**Comment by Regalis11** [comment]
You're right: radiation sickness applies a fixed amount of burns on each limb, so the number of limbs affects how fast the burn damage accumulates. In my opinion this is such a minor issue though that I think we can ignore it.
